deploy/redis/templates/clusterdefinition.yaml

```diff
@@ -99,6 +99,13 @@ spec:
               - name: REDIS_ARGS
                 value: "--requirepass $(REDIS_PASSWORD)"
             command: ["/scripts/redis-start.sh"]
+            lifecycle:
+              preStop:
+                exec:
+                  command:
+                    - /bin/bash
+                    - -c
+                    - "redis-cli -a $REDIS_DEFAULT_PASSWORD acl save"
```

Comment:
Here we need to be backwards compatible with scenarios where the $REDIS_DEFAULT_PASSWORD environment variable is not present in release-0.5 versions.

Reply:
added the logic to check  password existence.